perun-rpc/src/main/perl/listOfRequiredVoAttributes

```diff
@@ -22,7 +22,8 @@ sub help {
 	};
 }
 
-my ($voId, $voShortName, $sortingFunction, $batch);
+our $batch;
+my ($voId, $voShortName, $sortingFunction);
```

Comment:
API method requires also ``service`` property as you said, so please update this tool and call ``getRequiredAttributes(vo =>$voId);`` also with ``service => $serviceId`` param. Thank you.